app/errors/forest_liana/live_query_error.rb

```diff
@@ -0,0 +1,9 @@
+module ForestLiana
+  class LiveQueryError < StandardError
+    attr_reader :message
+
+    def initialize(message)
+      @message = message || 'Something went wrong'
+    end
+  end
+end
```

Comment:
There is already a custom error definition `SerializeAttributeBadFormat` in `config/initializers/errors`.

Reply:
I think you should follow the existing code convention.